WIP: Feature/environment preconfiguration

patrick-skowronek opened the pull request:
**What's in the PR**
* Moving environment variables to the docker image
With this PR many variables moved to the image and should be overwritten with a docker.override

**How to test manually**
* ...

torbrenner committed:
feature: Added default variables per project build

patrick-skowronek committed:
Moved environment variables to docker image

patrick-skowronek committed:
Fixed missing postgres user

patrick-skowronek committed:
Add missing MDR to gbn and correct POSTGres PW

patrick-skowronek committed:
Add missing id-manager url to dktk

alexanderkiel commented on .github/workflows/ci.yml:
```diff
@@ -8,6 +8,7 @@ on:
     branches:
       - master
       - develop
+      - feature/environmentPreconfiguration
```
This is not needed. If you file a PR, the branch will be build.

patrick-skowronek committed:
Added missing Var

patrick-skowronek committed:
Merge remote-tracking branch 'origin/release-7.6.1' into feature/environmentPreconfiguration

patrick-skowronek committed:
Added config for forward proxy

patrick-skowronek committed:
Fixed naming of services

patrick-skowronek committed:
Added NNGM Docker image

patrick-skowronek committed:
Merge branch 'master' into feature/environmentPreconfiguration

patrick-skowronek committed:
Changed nngm profile due to no ldm needed

patrick-skowronek committed:
Build nngm image as dktk

patrick-skowronek committed:
finalized nngm docker image

patrick-skowronek committed:
Merge branch 'master' into feature/environmentPreconfiguration

patrick-skowronek committed:
Update Dockerfile

Updated proxy settings

Commit:
Fixed Blaze URL for GBN

The Blaze URL was missing the trailing "/fhir", which prevented the
Connector from executing queries. This patch fixes the problem.

DavidCroftDKFZ committed:
Merge pull request #177 from samply/feature/environmentPreconfiguration__fix_blaze_url

Fixed Blaze URL for GBN
